cmd/krew/cmd/root.go

```diff
@@ -86,6 +104,17 @@ func init() {
 }
 
 func preRun(cmd *cobra.Command, _ []string) error {
+	go func() {
+		if _, ok := os.LookupEnv("KREW_NO_UPGRADE_CHECK"); ok {
+			return
+		}
+		var err error
+		latestTag, err = updatecheck.LatestTag()
+		if err != nil {
+			klog.Warning(err)
```

Comment:
I suspect this directly goes to users. Let's not do that.
V(1).Infof("WARNING:") is better.